kinesis_video_streamer/test/kinesis_video_streamer_test_utils.h

```diff
@@ -21,9 +21,12 @@
 #include <kinesis_video_streamer/streamer.h>
 
 using namespace std;
+using namespace com::amazonaws::kinesis::video;
```

Comment:
Could we avoid adding new using namespaces or clean this up to remove using namespace from header files. I realize this is a test, however it can definitely get confusing later.

Reply:
This is a unit test header, which I did not impose the `using namespace` ban for sake of time.